src/igvfd/searches/configs/prediction.py

```diff
@@ -16,10 +16,42 @@ def prediction():
             'lab.title': {
                 'title': 'Lab'
             },
+            'prediction_type': {
```

Comment:
prediction_type duplicated here

Reply:
Can you also remove trailing comma L76, and empty L78

Reply:
fixed